arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -368,6 +368,10 @@
           "description": "The uri for the swagger spec associated with this web service.",
           "readOnly": true
         },
+        "exposeSampleData": {
+            "type": "boolean",
+            "description": "If true, the WebServiceProperties will expose sample data. Otherwise, it will not have it."
```

Comment:
The description should say  "Flag that controls whether to expose sample data or not in the web service's swagger definition